test/FrontendContextualise/Module/Open.hs

```diff
@@ -0,0 +1,30 @@
+module FrontendContextualise.Module.Open where
+
+import Juvix.Library
+import qualified Juvix.FrontendContextualise.ModuleOpen.Environment as Env
+import qualified Data.Text as Text
+import qualified Juvix.Core.Common.Context as Context
+import qualified Juvix.Core.Common.NameSpace as NameSpace
+import qualified Juvix.Library.HashMap as HashMap
+-- import qualified Test.Tasty as T
+-- import qualified Test.Tasty.HUnit as T
+
```

Comment:
needs tasty tests?

Reply:
yes, but that will come in a bit, it's commented out atm, as it's more convinent to add it back in the import when I actually do use them